Documentation/ApiOverview/FlexForms/Index.rst

```diff
@@ -370,40 +370,51 @@ You can use the :php:`FlexFormService` to read the content of a FlexForm field:
     use TYPO3\CMS\Core\Service\FlexFormService;
     use TYPO3\CMS\Core\Utility\GeneralUtility;
 
-    // Inject FlexFormService
-    public function __construct(
-        protected readonly FlexFormService $flexFormService,
-    ) {
-    }
-
-    private function loadFlexForm(): void
+    class NonExtbaseController
     {
-        $this->flexFormData = $this->flexFormService
-            ->convertFlexFormContentToArray($this->cObj->data['pi_flexform']);
+
+        // Inject FlexFormService
+        public function __construct(
+            private readonly FlexFormService $flexFormService,
+        ) {
+        }
+
+        // ...
+
+        private function loadFlexForm($flexFormString): array
+        {
+            return $this->flexFormService
+                ->convertFlexFormContentToArray($flexFormString);
+        }
     }
 
-The resulting array would look like this:
+Using :php:`FlexFormService->convertFlexFormContentToArray` the resulting
+array can be used conviniently in most use cases:
 
 ..  code-block:: php
-    :caption: Example output
 
-    $output = $this->flexFormService
-            ->convertFlexFormContentToArray($this->cObj->data['pi_flexform']);
+     var_export($this->flexFormService
```

Comment:
Aaah, now.... can we format that better? Also: [var_export](https://www.php.net/manual/de/function.var-export.php#refsect1-function.var-export-examples) shows `array( ... )` not `[]` - nitpicking

Reply:
yeah I know but it is much easier to read this way. This is not about var_export. You are a little bit to nit picky here I think.

No idea how to format that better but go ahead if you have an idea

Reply:
Yup, want to just say it ;-)

I meant that with formatting:
```
var_export(
    $this->flexFormService->convertFlexFormContentToArray($flexFormString)
);
```

reads better, from my side you can also write everything in one line. Maybe just run the cs fixer, I assume it will format it somehow ;-).